src/app/board/board.component.html

```diff
@@ -1,10 +1,10 @@
 <div id="board_topWorkItems" class="container-fluid container-cards-pf">
-  <h3>Top WorkItems</h3>
-  <div class="grid grid-pad">
-    <div *ngFor="let workItem of workItems; let count = index" (click)="gotoDetail(workItem)" class="col-1-4" id="{{'board_topWorkItemList_'+count}}">
-      <div class="module workItem">
-        <h4>{{workItem.fields['system.title']}}</h4>
+  <div class='board'>
+      <div class='lane' *ngFor='let lane of lanes'>
+          <div class="lane-header">{{lane.title}}</div>
+          <div [dragula]="'nested-bag'">
+            <div *ngFor='let card of lane.cards' class="card" (click)="gotoDetail(card)">{{card.fields['system.title']}}</div>
```

Comment:
@harrybabu I think we need some IDs and/or Classes here so that the functional tests can find it in the UI. Take a look at how it is done in the List-view. @ldimaggi can you provide input here too?